lib/webview-helpers.js

```diff
@@ -16,23 +18,36 @@ let helpers = {};
 // same as chrome-backed webviews)
 // TODO: some of this function belongs in appium-adb
 async function webviewsFromProcs (adb, deviceSocket) {
+  const WEBVIEW_REGEXP = `@?webview_devtools_remote_(\\d+)`;
```

Comment:
Why not just make this a RegExp object and skip the step in line 38. It could also go above wth the rest of the constants.